specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2015-04-08/cosmos-db.json

```diff
@@ -1443,9 +1443,9 @@
         }
       }
     },
-    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DocumentDB/databaseAccounts/{accountName}/apis/sql/databases": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DocumentDB/databaseAccounts/{accountName}/sqlDatabases": {
```

Comment:
this is breaking change. pls add a new api version.

Reply:
Hi, How can I add a new api version?

Reply:
add a new swagger file with new tag under azure-rest-api-specs/specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable or preview, such as 2019-07-01 etc. sample azure-rest-api-specs/specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2015-04-08/

Reply:
Hi @yungezz my alias is chuyu. Can you please ping me, let's sync on Teams.